sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -1147,4 +1146,6 @@ load/store overloads
 |11  |2024-04-29 |Yury Plyakhin | Add 1x64x16 supported combination for
 Intel XMX (intel_gpu_pvc)
 |12  |2024-06-14 |Jack Kirk | Add note on sm version device matching issue.
-|======================
+|13  |2024-08-07 |Yixing Zhang | Add 32x32x16 supported combination for
```

Comment:
No need to add this here.
This is supposed to track new features.
It should not become laundry list of every commit we do to the file.

Reply:
Thanks for the suggestion! :) I have removed it